homeassistant/const.py

```diff
@@ -147,6 +147,8 @@
 SERVICE_MOVE_DOWN = 'move_down'
 SERVICE_STOP = 'stop'
 
+SERVICE_SET_OPTION = 'set_option'
```

Comment:
No need to put it in the `const` file, just keep it inside the component as this will not be reused by other components.